src/index.html

```diff
@@ -4,10 +4,308 @@
+              src="./images/people/1.png"
+              alt="Leslie Mckinney photo"
+            >
+
+            <article class="reviews__info">
+              <h3 class="reviews__second-title">Leslie Mckinney</h3>
+              <p class="reviews__profession">CEO</p>
+              <p class="reviews__review">
+                Nice work, Certificates National Association of Realtors (USA)
+              </p>
+            </article>
+
+          </div>
+
+          <div class="reviews__block">
+            <img
+              class="reviews__photo"
+              src="./images/people/2.png"
+              alt="Leslie Mckinney photo"
+            >
+
+            <article class="reviews__info">
+              <h3 class="reviews__second-title">Scarlett Hawkins</h3>
+              <p class="reviews__profession">Senior Vice President</p>
+              <p class="reviews__review">
+                My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+              </p>
+            </article>
+
+          </div>
+
+          <div class="reviews__block">
+            <img
+              class="reviews__photo"
+              src="./images/people/3.png"
+              alt="Leslie Mckinney photo"
+            >
+
+            <article class="reviews__info">
+              <h3 class="reviews__second-title">Max Henry</h3>
+              <p class="reviews__profession">Chief Design Officer</p>
+              <p class="reviews__review">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </p>
+            </article>
+
+          </div>
+
+        </div>
+
+      </section>
+
+      <section class="main__contact-us contact-us" id="contacts">
+        <h2 class="contact-us__titile">Contat us</h2>
+        <div class="contact-us__mini-logo"></div>
+
+        <div class="contact-us__container">
+          <div class="contact-us__contacts">
+            <div class="contact-us__contact">
+              <p class="contact-us__about-link">Phone</p>
+              <a
+                class="contact-us__link"
+                href="tel:+1234-555-55-55"
+              >
+                +1 234 555-55-55
+              </a>
+              <p class="contact-us__about-link">Email</p>
+              <a
+                class="contact-us__link"
+                href="mailto: hello@miami.com"
+              >
+               hello@miami.com
+              </a>
+            </div>
+            <div class="contact-us__adress">
+              <p class="contact-us__about-link">Address</p>
+              <a
+                class="contact-us__link"
+                href="https://goo.gl/maps/1NdKrmvzA8eHz1s57"
+              >
+              400 first ave. n. suite 700 Minneapolis, MN 55401
+              </a>
+            </div>
+          </div>
+
+          <form
+            class="contact-us__consultation"
+            method="POST"
+          >
```

Comment:
```suggestion
          <form
            action="#"
            class="contact-us__consultation"
            method="POST"
          >
```
Add `action` attribute to avoid error when submitting your form